pkg/controller/volume/persistentvolume/scheduler_binder.go

```diff
@@ -263,18 +268,26 @@ func (b *volumeBinder) AssumePodVolumes(assumedPod *v1.Pod, nodeName string) (al
 
 // BindPodVolumes gets the cached bindings and PVCs to provision in podBindingCache
 // and makes the API update for those PVs/PVCs.
-func (b *volumeBinder) BindPodVolumes(assumedPod *v1.Pod) error {
+func (b *volumeBinder) BindPodVolumes(assumedPod *v1.Pod) (err error) {
 	podName := getPodName(assumedPod)
 	glog.V(4).Infof("BindPodVolumes for pod %q", podName)
 
+	start := time.Now()
+	defer func() {
+		VolumeSchedulingStageLatency.WithLabelValues("bind").Observe(SinceInSeconds(start))
```

Comment:
`Observe` is called when the `BindPodVolumes` returns an error as well. Is this intended or `Observe` should be called only if `err` is not nil?

Reply:
intended